command/command_test.go

```diff
@@ -236,6 +259,10 @@ Error: fourth error`,
 				t.Errorf("command.Run() error = %v, wantErr %v", gotErrMsg, tt.wantErr)
 				return
 			}
+			gotFormattedMsg := errorutil.FormattedError(err)
+			if gotFormattedMsg != tt.wantErr {
```

Comment:
gotFormattedMsg shouldn't be the same as gotErrMsg.

gotErrMsg should be (`<error>: <reason>`):

```
command failed with exit status 1 (bash "testdata/exit_with_message.sh"): Error: first error
Error: second error
Error: third error
Error: fourth error
```

while gotFormattedMsg should be `<error>:\n\t<reason>`: 

```
command failed with exit status 1 (bash "testdata/exit_with_message.sh"): 
  Error: first error
  Error: second error
  Error: third error
  Error: fourth error
```

Reply:
Added a new test (only slightly janky) in the errorutil package.